pkg/reconciliation/reconcile_racks.go

```diff
@@ -2092,16 +2094,152 @@ func (rc *ReconciliationContext) CheckConditionInitializedAndReady() result.Reco
 	return result.Continue()
 }
 
-func (rc *ReconciliationContext) cleanupAfterScaling() error {
-	var err error
+const (
+	podJobIdAnnotation     = "cassandra.datastax.com/job-id"
+	podJobStatusAnnotation = "cassandra.datastax.com/job-status"
+	podJobHandler          = "cassandra.datastax.com/job-runner"
+	jobHandlerMgmtApi      = "management-api"
+
+	podJobCompleted = "COMPLETED"
+	podJobError     = "ERROR"
+	podJobWaiting   = "WAITING"
+)
+
+var (
+	jobRunner chan int = make(chan int, 1)
+)
+
+func (rc *ReconciliationContext) cleanupAfterScaling() result.ReconcileResult {
+	// We sort to ensure we process the pods in the same order
+	sort.Slice(rc.dcPods, func(i, j int) bool {
```

Comment:
If a pod is deleted and recreated then the order changes. It may not be a big deal but to keep things consistent and predictable, what do you think about sort first by rack and then by pod name? We wouldn't need to use the rack for just for sorting purposes, it helps ensure do one rack at a time.

Reply:
Well, I added now an option that sorts by rack and by name. Would that work?